data/campaigns/World_Conquest/lua/game_mechanics/effects.lua

```diff
@@ -66,12 +66,12 @@ end
 -- Like apply_to=resistance with replace=true, but never decreases resistances.
 function wesnoth.effects.wc2_min_resistance(u, cfg)
 	local resistance_new = {}
-	local resistance_old = wml.parsed(wml.get_child(cfg, "resistance"))
+	local resistance_old = wml.get_child(cfg, "resistance")
```

Comment:
I think the `wml.parsed` should be left in here, as this is an `[effect]`.

Reply:
I have restored this instance of `wml.parsed` as well.